.github/workflows/playwright.yml

```
@@ -17,6 +17,18 @@ jobs:
         run: |
           echo "VITE_GOOGLE_MAPS_API_KEY=${{secrets.GOOGLE_MAPS_KEY}}" >> .env
           echo "VITE_GMAPS_VERSION=3.58" >> .env
+      - name: Determine cache directory
+        id: npm-cache-dir
+        run: echo "dir=$(npm config get cache)" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:39: Double quote to prevent globbing and word splitting

(shellcheck)